irods/test/data_obj_test.py

```diff
@@ -131,6 +131,37 @@ def test_move_obj_to_coll(self):
         # remove new collection
         new_coll.remove(recurse=True, force=True)
 
+    def test_copy_existing_obj_to_relative_dest_fails_145(self):
```

Comment:
update these test names (x3) from `_145` to `_irods4796` and we'll merge it

Reply:
done.